spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -22,6 +23,49 @@
 
 
 class SpotSkillManager:
+    """
+    Class to manage the skills for Spot
```

Comment:
Maybe change to: "Interface class to invoke skills for Spot, also exposes..."

"manage the skills" reads pretty vague.

Reply:
Addressed